packages/next/src/shared/lib/image-blur-svg.ts

```diff
@@ -16,20 +16,21 @@ export function getImageBlurSvg({
   blurDataURL: string
   objectFit?: string
 }): string {
-  const std = blurWidth && blurHeight ? '1' : '20'
+  const std = blurWidth && blurHeight ? 0.5 : 20
   const svgWidth = blurWidth || widthInt
   const svgHeight = blurHeight || heightInt
-  const feComponentTransfer = blurDataURL.startsWith('data:image/jpeg')
-    ? `%3CfeComponentTransfer%3E%3CfeFuncA type='discrete' tableValues='1 1'/%3E%3C/feComponentTransfer%3E%`
+
+  const svgDimensions = svgWidth && svgHeight
+  const svgSizeAttributes = svgDimensions
+    ? `viewBox='0 0 ${svgWidth} ${svgHeight}'`
     : ''
-  if (svgWidth && svgHeight) {
-    return `%3Csvg xmlns='http%3A//www.w3.org/2000/svg' viewBox='0 0 ${svgWidth} ${svgHeight}'%3E%3Cfilter id='b' color-interpolation-filters='sRGB'%3E%3CfeGaussianBlur stdDeviation='${std}'/%3E${feComponentTransfer}%3C/filter%3E%3Cimage preserveAspectRatio='none' filter='url(%23b)' x='0' y='0' height='100%25' width='100%25' href='${blurDataURL}'/%3E%3C/svg%3E`
-  }
-  const preserveAspectRatio =
-    objectFit === 'contain'
-      ? 'xMidYMid'
-      : objectFit === 'cover'
-      ? 'xMidYMid slice'
-      : 'none'
-  return `%3Csvg xmlns='http%3A//www.w3.org/2000/svg'%3E%3Cimage style='filter:blur(20px)' preserveAspectRatio='${preserveAspectRatio}' x='0' y='0' height='100%25' width='100%25' href='${blurDataURL}'/%3E%3C/svg%3E`
+  const preserveAspectRatio = svgDimensions
+    ? 'none'
+    : objectFit === 'contain'
+    ? 'xMidYMid'
+    : objectFit === 'cover'
+    ? 'xMidYMid slice'
+    : 'none'
+
+  return `%3csvg xmlns='http://www.w3.org/2000/svg' ${svgSizeAttributes}%3e%3cfilter id='b' color-interpolation-filters='sRGB'%3e%3cfeGaussianBlur stdDeviation='${std}'/%3e%3cfeColorMatrix values='1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 100 -1' result='s'/%3e%3cfeFlood x='0' y='0' width='100%25' height='100%25'/%3e%3cfeComposite operator='out' in='s'/%3e%3cfeComposite in2='SourceGraphic'/%3e%3cfeGaussianBlur stdDeviation='${std}'/%3e%3c/filter%3e%3cimage width='100%25' height='100%25' x='0' y='0' preserveAspectRatio='${preserveAspectRatio}' style='filter: url(%23b);' href='${blurDataURL}'/%3e%3c/svg%3e`
```

Comment:
```suggestion
  return `%3csvg xmlns='http://www.w3.org/2000/svg' ${viewBox}%3e%3cfilter id='b' color-interpolation-filters='sRGB'%3e%3cfeGaussianBlur stdDeviation='${std}'/%3e%3cfeColorMatrix values='1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 100 -1' result='s'/%3e%3cfeFlood x='0' y='0' width='100%25' height='100%25'/%3e%3cfeComposite operator='out' in='s'/%3e%3cfeComposite in2='SourceGraphic'/%3e%3cfeGaussianBlur stdDeviation='${std}'/%3e%3c/filter%3e%3cimage width='100%25' height='100%25' x='0' y='0' preserveAspectRatio='${preserveAspectRatio}' style='filter: url(%23b);' href='${blurDataURL}'/%3e%3c/svg%3e`
```

I think `viewBox` is a better name than `svgSizeAttributes`